decentraland/unity-renderer
# Fix/procedural skybox satellite movement

## What does this PR change?



Fix: Satellite movement was not smooth. Corrected in this branch

## How to test the changes?



1. Go to: https://play.decentraland.org/?island=Ijod&position=1%2C-1&realm=loki&renderer=https%3A%2F%2Fdecentraland-olavra.s3.eu-west-1.amazonaws.com%2Fbranch%2Ffix%2Fprocedural-skybox-satellite-movemnt
2. Can check the movement of sun and the moon for smooth transitions.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
